src/Compute/Compute/StorageServices/AddAzureVhdCommand.cs

```diff
@@ -378,7 +394,8 @@ private PSDisk CreateDiskConfig()
                 EncryptionSettingsCollection = null,
                 Encryption = null,
                 NetworkAccessPolicy = null,
-                DiskAccessId = null
+                DiskAccessId = null,
+                DataAccessAuthMode = this.IsParameterBound(c => c.DataAccessAuthMode) ? this.DataAccessAuthMode : null
```

Comment:
is there differences between
DataAccessAuthMode = "None"
and
DataAccessAuthMode = null

Reply:
there is no difference. However I believe they did not pursue switch parameter in case they add other DataAccessAuthMode in the future

Reply:
you don't have to make it switch parameter, just delete the "None" value and add more values when needed in the future